itest/lnd_open_channel_test.go

```diff
@@ -445,6 +463,14 @@ func runBasicChannelCreationAndUpdates(ht *lntest.HarnessTest,
 		}
 	}
 
+	// If Bob now tries to open a channel with an invalid memo, reject it.
+	invalidMemo := strings.Repeat("a", 501)
+	ht.OpenChannelAssertErr(ht.Bob, ht.Alice, lntest.OpenChannelParams{
+		Amt:  funding.MaxBtcFundingAmount,
+		Memo: invalidMemo,
+	}, fmt.Errorf("provided memo (%s) is of length 501, exceeds 500",
```

Comment:
nit: weird formatting. Maybe declare the `expectedErr` first, then pass it into the `OpenChannelAssertErr` function?

Reply:
broke out the channel params as well to aid readability